ios/safari_extension/SafariWebExtensionHandler.swift

```diff
@@ -0,0 +1,19 @@
+import SafariServices
+import os.log
+
+let SFExtensionMessageKey = "message"
+
+class SafariWebExtensionHandler: NSObject, NSExtensionRequestHandling {
+
+    func beginRequest(with context: NSExtensionContext) {
+        let item = context.inputItems[0] as! NSExtensionItem
+        let message = item.userInfo?[SFExtensionMessageKey]
+        os_log(.default, "Received message from browser.runtime.sendNativeMessage: %@", message as! CVarArg)
```

Comment:
I figure we can do slightly better than Apple’s template by using the modern logging API 😁

```suggestion
		Logger().log("Received message from browser.runtime.sendNativeMessage: \(String(describing: message))")
```